### ISSUE TITLE
Fix 'test_feeds/test_invalid_type_custom_feeds' tests in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1543]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1531|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Vulnerability Detector
Support Team:  "binary beasts"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

|  Type | Description  |
|--|--|
|Test Path| test_feeds/test_invalid_type_custom_feeds
| Consistent | No 
| Test Execution | 1/3 executions failed
| Cases |   test_invalid_type_custom_feeds
| Cases Fails | 24 
| Summary | On `test_invalid_type_custom_feeds` when `check_feed_imported_successfully` method is called. The problem seems to be a Parser error when imported the file. 
| Error message |  "Could not find the message: '{log_system_name}' feed finished successfully"

In order to finish this issue the following tasks should be fulfilled:
- [ ] Research of fails.
- [ ] Apply Fix on the Test.
- [ ] Full Green/ Full Yellow  in `test_feeds/test_invalid_type_custom_feeds` for 3 times.
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by mdengra** [comment]
### 2021-07-13
After multiple runs, I have not been able to replicate the error on my testing environment:

**Comment by damarisg** [comment]
This issue requires more research because when we execute it, we keep getting fails.  We can see that this test is inconsistency.

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[Results1_InvalidCustom.log](https://github.com/wazuh/wazuh-qa/files/6810919/Results1_InvalidCustom.log) | 2021-07-13| Seyla | 🟢
| [Results2_InvalidCustom.log](https://github.com/wazuh/wazuh-qa/files/6810918/Results2_InvalidCustom.log) | 2021-07-13| Seyla | 🔴
|[Results3_InvalidCustom.log](https://github.com/wazuh/wazuh-qa/files/6810941/Results3_InvalidCustom.log) | 2021-07-13| Seyla | 🟢

**Comment by pereyra-m** [comment]
Results for `test_feeds/test_invalid_type_custom_feeds`.
It is important to know that both green rounds have **XFAIL**.


| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[centos_test_invalid_type_custom_feeds_1.log](https://github.com/wazuh/wazuh-qa/files/6812373/centos_test_invalid_type_custom_feeds_1.log)| 2021-07-13| Matias | :red_circle:
| [centos_test_invalid_type_custom_feeds_2.log](https://github.com/wazuh/wazuh-qa/files/6812372/centos_test_invalid_type_custom_feeds_2.log)| 2021-07-13| Matias | :green_circle:
| [centos_test_invalid_type_custom_feeds_3.log](https://github.com/wazuh/wazuh-qa/files/6812371/centos_test_invalid_type_custom_feeds_3.log)| 2021-07-13| Matias | :green_circle:

**Comment by pereyra-m** [comment]
Results for `test_feeds/test_invalid_type_custom_feeds`.
Now, the following modules were turned off: `sca`, `syscollector` and `rootcheck`
It is important to know that both green rounds have **XFAIL**.







| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_invalid_type_custom_feeds_modules_off_1.log](https://github.com/wazuh/wazuh-qa/files/6816205/test_invalid_type_custom_feeds_modules_off_1.log)| 2021-07-14| Matias | :green_circle:
| [test_invalid_type_custom_feeds_modules_off_2.log](https://github.com/wazuh/wazuh-qa/files/6816204/test_invalid_type_custom_feeds_modules_off_2.log)| 2021-07-14| Matias | :green_circle:
| [test_invalid_type_custom_feeds_modules_off_3.log](https://github.com/wazuh/wazuh-qa/files/6816203/test_invalid_type_custom_feeds_modules_off_3.log)| 2021-07-14| Matias | :green_circle:

**Comment by mdengra** [comment]
### 2021-07-14
I ran the tests again by recreating a new Vagrant box with a clean environment.

Test results with the default modules enabled in the `ossec.conf`: 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_invalid_type_custom_feeds_local_r1.log](https://github.com/wazuh/wazuh-qa/files/6816745/test_invalid_type_custom_feeds_local_r1.log)| 2021-07-14| Miguel | :green_circle: 
|[test_invalid_type_custom_feeds_local_r2.log](https://github.com/wazuh/wazuh-qa/files/6816746/test_invalid_type_custom_feeds_local_r2.log)| 2021-07-14| Miguel | :green_circle: 
|[test_invalid_type_custom_feeds_local_r3.log](https://github.com/wazuh/wazuh-qa/files/6816752/test_invalid_type_custom_feeds_local_r3.log)| 2021-07-14| Miguel | :green_circle:

**Comment by damarisg** [comment]
Results when disabling modules: `sca`, `syscollector` and `rootcheck`.


| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[InvalidType1.log](https://github.com/wazuh/wazuh-qa/files/6818333/InvalidType1.log)| 2021-07-14| Seyla| 🟡 
| [InvalidType2.log](https://github.com/wazuh/wazuh-qa/files/6818334/InvalidType2.log)| 2021-07-14| Seyla| :green_circle:
| [InvalidType3.log](https://github.com/wazuh/wazuh-qa/files/6818332/InvalidType3.log)| 2021-07-14| Seyla| 🟡 


| Reference | Status | 
|--|--|
| 🟢 | Pass without warnings|
| 🟡 | Pass with Warnings |

**Comment by damarisg** [comment]
This issue is closed, after executes for a lot of times and see that works. The solved applied was disabling `sca`, `syscollector` and `rootcheck` modules.

**Comment by damarisg** [comment]
This issue is closed by #1605 

## Test Results from `1531-full-yellow-vuln-det`

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[YellowResultsInvalidTypeCustomFeed.log](https://github.com/wazuh/wazuh-qa/files/6856171/YellowResultsInvalidTypeCustomFeed.log)| 2021-07-21| Seyla| 🟡 
| [YellowResultsInvalidTypeCustomFeed2.log](https://github.com/wazuh/wazuh-qa/files/6856175/YellowResultsInvalidTypeCustomFeed2.log)| 2021-07-21| Seyla| 🟡 
| [YellowResultsInvalidTypeCustomFeed3.log](https://github.com/wazuh/wazuh-qa/files/6856179/YellowResultsInvalidTypeCustomFeed3.log)| 2021-07-21| Seyla| 🟡 


| Reference | Status | 
|--|--|
| 🟡 | Pass with Warnings |

**Comment by damarisg** [comment]
This test works, but when it is executed with all Vulnerability tests sometime fail, The problem is the [FileMonitor](https://github.com/wazuh/wazuh-qa/issues/1602) for this reason I reopen and blocked this test.

**Comment by damarisg** [comment]
This issue is closed because the test was disabled by #1602
